prefs.js

```diff
@@ -59,6 +59,33 @@ export default class TogglerPreferences extends ExtensionPreferences {
     rowShortcut.add_suffix(entryShortcut);
     rowShortcut.activatable_widget = entryShortcut;
 
+    // Workspaces
+    const rowWorkspaces = new Adw.ActionRow({
+      title: "Workspaces Mode",
+      subtitle: "What to do when terminal window is in different workspace"
+    })
+    group.add(rowWorkspaces)
+
+    const workspaceModes = new Gtk.StringList()
+    workspaceModes.append("Switch to opened")
+    workspaceModes.append("Move into space")
+    workspaceModes.append("New for workspace")
```

Comment:
Perhaps we could move this into the header right under imports, and make it a bit more descriptive + since I want to make _Move the window to the current workspace the default one_, make it go first.

```js
const workspaceModes = [
  "Move the window to the current workspace",
  "Switch to the workspace with the terminal window",
  "Launch a new instance in the current workspace",
];
```